### DISCUSSION TITLE
Is it safe to update alias.email?

### DISCUSSION THREAD

**Discussion by Metabaron1** [simple-login/app#1648] [Self-hosting question] [answered]
Hi 
Dot ('.') was not allowed in alias email in initial versions. As it's now allowed, I would like to update old aliases without recreate new ones and losing history.
So, is it safe to manually update alias.email in database before starting sl-db or is there some side effects?

**Comment by nguyenkims** [comment, marked as answer]
The alias address is only stored in the alias table so the update is safe. However if you do that, a sender can no longer reach out to your old alias address right?

**Comment by Metabaron1** [comment]
Yes, before I will update the address at the sender.
Thanks for reply
